query/src/test/scala/filodb/query/exec/rangefn/AggrOverTimeFunctionsSpec.scala

```diff
@@ -146,20 +222,39 @@ class AggrOverTimeFunctionsSpec extends RawDataWindowingSpec {
   val numIterations = 10
 
   it("should correctly aggregate sum_over_time using both chunked and sliding iterators") {
-    val data = (1 to 240).map(_.toDouble)
```

Comment:
I see that new data part was added to existing test case.  Can we please separate out the new data to a new test case?  Something about checking for NaN's.